web/src/app/candidate/candidate.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import { isNullOrUndefined } from 'util';
+
+// Services
+import { CandidateServiceApp } from './shared/candidate.serviceApp';
+import { OpeningServiceApp } from '../opening/shared/opening.serviceApp';
+import { QualificationsServiceApp } from '../admin/qualifications/shared/qualifications.serviceApp';
+
+// Models
+import { OpeningViewModel } from '../webapi/models/opening-view-model';
+import { CandidateViewModel } from '../webapi/models';
+import { QualificationViewModel } from '../webapi/models/qualification-view-model';
+import { Status } from '../app.enum';
+
+@Component({
+    selector: 'app-candidate',
+    templateUrl: 'candidate.component.html'
+})
+
+export class CandidateComponent implements OnInit {
+
+    candidateModel: CandidateViewModel = {} as CandidateViewModel;
+    openings: OpeningViewModel[] = [] as OpeningViewModel[];
+    qualifications: QualificationViewModel[] = [] as QualificationViewModel[];
+    years: number[] = [];
+    months: number[] = [];
+
+    constructor(private openingServiceApp: OpeningServiceApp, private candidateServiceApp: CandidateServiceApp,
+        private qualificationServiceApp: QualificationsServiceApp, private route: ActivatedRoute,
+        private router: Router) {
+        this.years = Array(30).fill(0).map((x, i) => i);
+        this.months = Array(12).fill(0).map((x, i) => i);
+    }
+
+    ngOnInit(): void {
+        this.initializeMethods();
+    }
+
+    initializeMethods() {
+        this.getOpenings();
+        this.getCandidateById();
+        this.getAllQualifications();
```

Comment:
make folk join of opening and qualifications

Reply:
Done